modules/ROOT/pages/6.0-release-notes-schema.adoc

```diff
@@ -0,0 +1,38 @@
+= TinyMCE 6.0 API changes
+:navtitle: TinyMCE 6.0 API changes
+:description: TinyMCE 6.0 API changes
+:keywords: releasenotes, apis
+
+// TODO: Is the above accurate for this page?
```

Comment:
Does this need doing before we merge? I'm guessing given the other changes @ltrouton has done probably not.